Tasks/Doroshevich/Task7/Task7_5.py

```diff
@@ -0,0 +1,33 @@
+d = {}
```

Comment:
not the best way to implement cache, since it will be reused for all decorated functions